Release Notes: 2020-12-17 notes (Operation Center dashboards) (#32)
SEKOIA-IO/documentation: docs/releases/2020-12-17.md

# 2020-12-17: Operation Center’s Configurable Dashboard System

**SEKOIA.IO’s Operation Center now brings a new dashboard mechanism, that is fully configurable and adaptable to all needs. This new feature is now enabled per default for all SEKOIA.IO customers.**

> Dashboard are composed of modular widgets that can be placed where you want. Widgets can be configured to specify the date range, applied filters, display, etc.

Reviewer: ```suggestion
Dashboards are composed of modular widgets that can be placed where you want. Widgets can be configured to specify the date range, applied filters, display, etc.
```

> SEKOIA.IO comes with a pre-configured dashboard that gives a synthetic view of the current community activity, either from a operationnal security perspective (risk level, number of alerts, etc.) and from an activity perspective (list of last posted comments, last created alerts, etc.).

Reviewer: ```suggestion
SEKOIA.IO comes with a pre-configured dashboard that gives a synthetic view of the current community activity, either from an operational security perspective (risk level, number of alerts, etc.) or from an activity perspective (list of last posted comments, last created alerts, etc.).
```

![Default Dashboard of SEKOIA.IO’s Operation Center](/assets/release-20201217/oc-dashboard-overview.png)

> All SEKOIA.IO users are able to create new dashboard that would fit their specific needs. It’s also possible to clone an existing dashboard.

Reviewer: ```suggestion
All SEKOIA.IO users are able to create new dashboards that fit their specific needs. It’s also possible to clone an existing dashboard.
```

## Provided Widgets To Monitor SEKOIA.IO’s Operation Center Activity

> SEKOIA.IO’s Operation Center provides a several widgets:

Reviewer: ```suggestion
SEKOIA.IO’s Operation Center provides several widgets:
```
